src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -624,6 +647,11 @@ static ucs_status_t uct_ib_mlx5_devx_md_open(struct ibv_device *ibv_device,
         goto err_free;
     }
 
+    status = uct_ib_mlx5_devx_query_lag(md, &lag_state);
+    if (status != UCS_OK) {
+        goto err_free;
+    }
```

Comment:
is it possible QUERY_LAG command is not supported on older FW / device?

Reply:
@yosefe OK, if not supported on older FW.device, than fall back to original implementation.